### DISCUSSION TITLE
Spell Checking Disabler

### DISCUSSION THREAD

**Discussion by ZahlenZ** [outline/outline#5339] [Ideas]
My team has been utilizing this tool for internal tech documentation. This means we are including lots of variable, table, database names throughout the documentation. When writing sql objects out like my.fun.name.table_thing it's always linted as being spelled wrong, which can lead the page to looking a little crazy.  However, this only shows up while "editing" a page, if you don't click into the page the linting doesn't appear.  

It would be nice if we could turn off any spell checking as when a page is littered with our.fun_names it's really not possible to find words that may actually be misspelled vs other things anyways.

**Comment by tommoor** [comment]
Spellcheck is disabled in all code blocks and marks for this reason, I'd recommend making sure your code is always in an appropriate code style and you won't see red squigglies.

**Comment by ZahlenZ** [comment]
We have tables that that outline source to target mappings and just in
general write prose that contains references to objects.
